### PR TITLE
Add Husky & Prettier

### PR DESCRIPTION
**Pull request by heyfirst**
Hi @bazsup, I would like to add husky for checking in pre-commit / pre-push. Would be nice that this project has this for make our developer experience better!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "depa-testing-frontend",
   "version": "0.1.0",
   "license": "MIT",
```
Comment (bazsup) [inline package.json:4]: This repo is not private

Comment (heyfirst) [reply package.json:4]: private package. not the private repo.

**Comment by bazsup** [comment]
@heyfirst 
It would be great if you add lint-staged to make pre commit faster.

For pre push, I think it unnecessary if we already have pre commit. Can you explain more why we should have pre push?

**Comment by heyfirst** [comment]
it's depend on your working process. but my though we should have `yarn build` in somewhere for checking this source able to build.

My experience is if we add `yarn build` for every each commit. It's take too long. So I just add it to before we push because at least you can make sure that the code you push is able to build 100%. By the way, prettier and test is require for every commit because it's not take time too long.

**Pull request merged**